pkgs/tools/typesetting/tex/tex-match/default.nix

```diff
@@ -0,0 +1,31 @@
+{ rustPlatform, fetchFromGitHub, gtk3, pkg-config, glib, lib }:
+
+let
+  version = "1.2.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "tex-match";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "zoeyfyi";
+    repo = "TeX-Match";
+    rev = "v${version}";
+    sha256 = "093wrw4pq1llaivc644r93g96wq5alvh93rdswr6p58zqkqw4zgg";
+    fetchSubmodules = true;
```

Comment:
Submodules found in the git repo seem related to windows builds. Are you sure they are needed for Darwin / Linux builds?